src/olav_espenes_ex/ex02/file_stats.py

```diff
@@ -0,0 +1,33 @@
+def char_counts(textfilename):
+    """Counts characters in a text.
+    """
+
+    with open(textfilename, 'r') as myfile:
+        data = myfile.read()
+        counter = {}
+        for element in data:
+            if ord(element) in counter:
+                counter[ord(element)] += 1
+            else:
+                counter[ord(element)] = 1
+        dict_a = {key: 0 for key in range(256)}
+        dict_a.update(counter)
+        list_a = list(dict_a.values())
```

Comment:
Nice thinking. But there's a better way and you may need to check on the suggested solution for that.